docs/mgmt/readme.md

```diff
@@ -0,0 +1,660 @@
+# Management plane SDK generator - how it works and its corresponding configurations
+
+## Resource hierarchy in management plane SDK
+
+For the basic management plane SDK concepts, please refer the detailed document [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/README.md#key-concepts) first.
+
+A resource is usually represented by three classes in the generated SDK: a `[Resource]Data` class, a `[Resource]Resource` class, and a `[Resource]Collection` class.
+
+### `[Resource]Data` class
+
+This class represents the "model" of the resource, and it corresponds to a model definition in the swagger, usually with the same name by trimming the word `Data` off.
+
+### `[Resource]Resource` class
+
+This class represents the a full resource client object which contains a `Data` property exposing the details as a `[Resource]Data` type.
+
+### `[Resource]Collection` class
+
+This class contains operations for a resource collection that belongs to a specific parent resource. It provides most of the logical collection operations.
+
+| Collection Behavior | Collection Method |
+| :--- | :--- |
+| Iterate/List | `GetAll()` |
+| Index | `Get(string name)` |
+| Add | `CreateOrUpdate(string name, [Resource]Data data)` |
+| Contains | `Exists(string name)` |
+| TryGet | `GetIfExists(string name)` |
+
+The `[Resource]Collection` class is designed to implement `IEnumerable<[Resource]Resource>` and `IAsyncEnumerable<[Resource]Resource>` interfaces, therefore the `GetAll()` method with no required parameters are required on this class for the generator. In rare cases, this rule is violated and a `list-exception` configuration will be required to temporarily solve this. See see [List exception](#list-exception) section for details.
+
+The parent resource of this resource will carry all the methods to get its child resources by returning its corresponding collection. For instance, on the resource of `VirtualNetworkResource`, you will find a method `GetSubnets` which returns a `SubnetCollection` to represent a collection of `SubnetResource`:
```

Comment:
consider use present tenses，remove 'will'

Reply:
I changed all the tenses in this article to be present tense